app/views/homes/show.html.erb

```diff
@@ -72,5 +72,12 @@
     </table>
   </p>
 
+  <!--
+    /// DO NOT REMOVE ///
+    /// This counter is from http://www.webestools.com/visitors-counter-free-number-unique-visitors-users-visits-statistics.html ///
+    /// DO NOT REMOVE ///
+  -->
+  <div style="text-align:center;"><script type="text/javascript" src="http://services.webestools.com/cpt_visitors/40122-13-5.js"></script></div><a href="http://www.webestools.com/" style="display:block;text-align:center;" title="Tools services webmasters counters generators scripts tutorials free"></a>
```

Comment:
I added the comment to _evoke an aesthetic_ of 12 year old copy-pasting totally unknown JS code in 1995. It sounds like it is perhaps less than evocative?